tools/testing/target_determination/heuristics/__init__.py

```diff
@@ -1,21 +1,22 @@
 from typing import List
 
+from tools.testing.target_determination.heuristics.correlated_with_historical_failures import (
+    CorrelatedWithHistoricalFailures,
+)
+from tools.testing.target_determination.heuristics.edited_by_pr import EditedByPR
+
 from tools.testing.target_determination.heuristics.interface import (
     HeuristicInterface as HeuristicInterface,
     TestPrioritizations as TestPrioritizations,
 )
 
 # Heuristics
-# from tools.testing.target_determination.heuristics.previously_failed_in_pr import (
-#     PreviouslyFailedInPR,
-# )
-# from tools.testing.target_determination.heuristics.edited_by_pr import EditedByPR
-# from tools.testing.target_determination.heuristics.correlated_with_historical_failures import (
-#     CorrelatedWithHistoricalFailures,
-# )
+from tools.testing.target_determination.heuristics.previously_failed_in_pr import (
+    PreviouslyFailedInPR,
+)
 
 HEURISTICS: List[HeuristicInterface] = [
-    # PreviouslyFailedInPR(),
-    # EditedByPR(),
-    # CorrelatedWithHistoricalFailures(),
+    PreviouslyFailedInPR(),
+    EditedByPR(),
+    CorrelatedWithHistoricalFailures(),
```

Comment:
As this is a big PR to roll out, what do you think about holding `CorrelatedWithHistoricalFailures` heuristic for now and just go ahead with `PreviouslyFailedInPR` and `EditedByPR`, which are existing features?  Monitoring it for a few days, then `CorrelatedWithHistoricalFailures` will follow

Reply:
`CorrelatedWithHistoricalFailures` is also an existing feature, I just gave it a new name. You'll see it called `_get_file_rating_tests` in `test_selections.py`

Reply:
Ah ok, I thought that Cat has reverted and still working on that.  Sounds good if that changes has been landed again.